examples/jeventdemo/source/src/org/kaaproject/kaa/demo/event/EventDemo.java

```diff
@@ -0,0 +1,143 @@
+import org.kaaproject.kaa.client.event.EventFamilyFactory;
+import org.kaaproject.kaa.client.event.EventListenersResolver;
+import org.kaaproject.kaa.client.event.FetchEventListeners;
+import org.kaaproject.kaa.client.event.registration.EndpointRegistrationManager;
+import org.kaaproject.kaa.client.event.registration.UserAttachCallback;
+import org.kaaproject.kaa.client.transact.TransactionId;
+import org.kaaproject.kaa.common.endpoint.gen.UserAttachResponse;
+import org.kaaproject.kaa.schema.sample.event.thermo.ChangeDegreeRequest;
+import org.kaaproject.kaa.schema.sample.event.thermo.CustomThermoEventClassFamily;
+import org.kaaproject.kaa.schema.sample.event.thermo.CustomThermoEventClassFamily.DefaultEventFamilyListener;
+import org.kaaproject.kaa.schema.sample.event.thermo.ThermostatInfoRequest;
+import org.kaaproject.kaa.schema.sample.event.thermo.ThermostatInfoResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EventDemo {
+
+
+    private static final Logger LOG = LoggerFactory.getLogger(EventDemo.class);
+    public static void main(String[] args) {
+        LOG.info("Event demo has been started");
+        doWork();
+        LOG.info("Event demo has been stopped");
+    }
+
+
+    public static void doWork() {
+        KaaClient kaaClient = Kaa.newClient(new DesktopKaaPlatformContext(),
+                new SimpleKaaClientStateListener());
+        kaaClient.start();
+
+        EndpointRegistrationManager registrationManager = kaaClient
+                .getEndpointRegistrationManager();
+
+        registrationManager.attachUser("userExternalId", "userAccessToken",
+                new UserAttachCallback() {
+                    @Override
+                    public void onAttachResult(UserAttachResponse response) {
+                        LOG.info("Attach response"
+                                + response.getResult());
+                    }
+                });
+
+        EventFamilyFactory eventFamilyFactory = kaaClient.getEventFamilyFactory();
+        CustomThermoEventClassFamily tecf = eventFamilyFactory
+                .getCustomThermoEventClassFamily();
+
+        List<String> FQNs = new LinkedList<>();
+        FQNs.add("org.kaaproject.kaa.schema.sample.event.thermo.ThermostatInfoRequest");
+        FQNs.add("org.kaaproject.kaa.schema.sample.event.thermo.ChangeDegreeRequest");
+
+        EventListenersResolver eventListenersResolver = kaaClient
+                .getEventListenerResolver();
+
+        eventListenersResolver.findEventListeners(FQNs,
+                new FetchEventListeners() {
+                    @Override
+                    public void onRequestFailed() {
+                        LOG.info("Request failed");
+                    }
+
+                    @Override
+                    public void onEventListenersReceived(
+                            List<String> eventListeners) {
+                        String output="Listeners received: ";
```

Comment:
Log lines separately. One for main message and count of listeners, and one for each event listener resecived